service/prometheus/service.go

```diff
@@ -0,0 +1,81 @@
+package prometheus
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+)
+
+const (
+	prometheusReloadPath = "/-/reload"
+)
+
+type Prometheus interface {
+	Reload() error
+}
+
+type Config struct {
+	Logger micrologger.Logger
+
+	PrometheusAddress string
```

Comment:
`PrometheusAddress` or just `Address`?

Reply:
I think the package is already pretty clear, so I feel that it's clear enough with `Address`.

Reply:
The first stutters. I would go with the latter. And then being consistent below.

Reply:
Vote for `Address`

Reply:
Done.